examples/basic_mysql_connection/index.js

```diff
@@ -12,9 +12,16 @@ var connection = mysql.createConnection({
 });
 
 connection.connect();
-
-connection.query('SELECT * FROM todos', function (error, results, fields) {
+let sql = ['SELECT COUNT(*) as todo-Items FROM todos',
+			'SELECT * FROM todos where done',
```

Comment:
Make sure to be consistent about your capitalisation. If you use `SELECT`, you should also use `WHERE` and not `where`.